lms/config.py

```diff
@@ -2,12 +2,19 @@
 
 import os
 from dataclasses import dataclass
-from typing import Callable
+from typing import Callable, List, Optional
 
-from pyramid.config import Configurator, aslist
+from pyramid.config import Configurator
+from pyramid.config import aslist as _aslist
 from pyramid.settings import asbool
 
 
+def aslist(value: Optional[str]) -> List[str]:
+    # pyramid's aslist doesn't accept None values
```

Comment:
This is a bit of hack but because we have a "system" to handle the settings I reckon is the best options.

For any other option I tried/consider I fear that I might introduce changes for values that might be None/Falsy now.

This, IMO, provides a better handling of list-like settings, particularly a better way to introduce them without having to coordinate setting up the environment variable with the deploy.